public/js/user/requestDom.js

```diff
@@ -0,0 +1,55 @@
+// const personalDetails = document.querySelector('.personal--details');
+const next = document.querySelector('.next--button');
+const {
+  pesonalSection, requestSection,
+  personalDetails, yourRequest, fullNameSpan, phoneNumberSpan, cityNameSpan, streetNameSpan, requestSubmit,
+} = querySelectors(['pesonalSection', 'requestSection', 'personalDetails', 'yourRequest', 'fullNameSpan', 'phoneNumberSpan', 'cityNameSpan', 'streetNameSpan', 'requestSubmit'], ['.pesonal--section', '.request--section', '.personal--details', '.your--request', '.full--name-span', '.phone--number-span', '.city--name-span', '.street--name-span', '.request--submit']);
+
+next.addEventListener('click', (e) => {
+  e.preventDefault();
+  const formData = new FormData(personalDetails);
+  const personalInformation = {};
+  formData.forEach((value, key) => {
+    personalInformation[key] = value.trim();
+  });
+  if (!personalInformation.fullName) {
+    fullNameSpan.textContent = 'must not empty';
+    return false;
+  } if (!personalInformation.phoneNumber) {
+    phoneNumberSpan.textContent = 'must not empty';
+    return false;
+  } if (!personalInformation.phoneNumber.match(/^[0-9]+$/)) {
```

Comment:
@Jamalat-shamallakh I usually recommend a less-strict regex for phone numbers: people will put in phone numbers in many different ways, sometimes with spaces, dots, dashes, plus signs for country code, and so on:
+1 123 456 7890
123-456-789
123.456.789

So you want to allow that (or add some text to the user telling them not to use special characters).
Also, I'm bad at regexes so I use this to test mine out a lot: https://regexr.com/